### ISSUE TITLE
Licence issue

### ISSUE THREAD

**Issue by AnnetteGill** [MicrosoftLearning/MS-100T00-Microsoft-365-Identity-and-Services#124]
# Module: 02
#### Step: 12

There is no licence in the tenant called "Microsoft 365 E5".  Holly should be given the "Office 365 E5" licence. (and also the "Enterprise Mobility + Security E5" licence, as stated.

**Comment by tfrink** [comment]
Thanks Annette. We're looking into this now. We have a team that reviews the lab steps periodically, and they did not catch this in their latest review from a few weeks back. I'm checking with them now to verify this issue.

**Comment by tfrink** [comment]
When the labs were originally created, the tenant that we used referenced the Office 365 E5 license. Then in summer 2021, we switched tenants, and the new tenant used the Microsoft 365 E5 license. This lab was updated to reflect that. However, we experienced some issues with that tenant, so later in 2021, we switched back to the original tenant that we used, but we didn't catch this change in the lab. So Annette, you are correct in that this license name must be changed. I have made this change as you suggested back to Office 365 E5.
